### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by clwluvw**
Create Apache License 2.0

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2020 CafeBazaar
```
Comment (clwluvw) [inline LICENSE]: @mehdy Are you okay with this part?

**Comment by mehdy** [comment]
Why not GPLv3?

**Comment by clwluvw** [comment]
> Why not GPLv3?

Is it okay for trademark use?!

**Comment by mehdy** [comment]
It's okay for what we do with it.

**Comment by clwluvw** [comment]
@mehdy Done.

**Pull request merged**